client/src/components/responsiveDrawer.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { Drawer } from '@material-ui/core';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import withStyles from '@material-ui/core/es/styles/withStyles';
+
+const styles = (theme) => ({
+  dragger: {
+    height: '100%',
+    width: '100%',
+    cursor: 'n-resize',
+    padding: '5px 5px 5px 5px',
+    borderTop: '1px solid #ddd',
+    position: 'absolute',
+    top: 0,
+    bottoms: 0,
+    // zIndex: '100',
+    backgroundColor: '#f4f7f9',
+  },
+});
+
+/**
+ * This class create a drawer that user can resize
+ */
+class ResponsiveDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isResizing: false,
+      lastDownY: 0,
+      newHeight: { height:300 },
+    };
+
+    this.handleMouseDown = this.handleMouseDown.bind(this);
+    this.handleMouseMove = this.handleMouseMove.bind(this);
+    this.handleMouseUp = this.handleMouseUp.bind(this);
+  }
+
+  handleMouseDown(e) {
+    this.setState({ isResizing:true, lastDownY:e.clientY });
+  }
+
+  handleMouseMove(e) {
+    if (!this.state.isResizing) {
+      return;
+    }
+
+    let offsetBottom = document.body.offsetHeight - (e.clientY - document.body.offsetTop);
+    let minHeight = 50;
+    let maxHeight = 600;
+    if (offsetBottom > minHeight && offsetBottom < maxHeight) {
+      this.setState({ newHeight:{ height:offsetBottom } });
+    }
+  }
+
+  handleMouseUp(e) {
+    this.setState({ isResizing:false });
+  }
+
+  componentDidMount() {
+    document.addEventListener('mousemove', (e) => this.handleMouseMove(e));
+    document.addEventListener('mouseup', (e) => this.handleMouseUp(e));
+  }
+
+  render() {
+    const { images } = this.props;
+    return (
+      images.length > 0 &&
+         images.map((thumbnail) =>
+
+             <img alt={'Image:' + thumbnail.id}
+               height='45'
```

Comment:
drawer height may need to be adjusted on resize

Reply:
ah. This is image height.

Reply:
I played with different images heights, I did not want to use the full height because for large images that might not look great, 45 seemed to be good for most images I tried cantilever, nanoparticle, and ellipse.

Reply:
Now the drawer uses the image height up to a max drawer height set in the embeddingMorseSmaleWindow.js